resources/api/api.raml

```diff
@@ -1109,6 +1111,56 @@ documentation:
                application/json:
                  example: !include examples/virhe-response.json
                  schema: virhe-response
+    /reikapaikkaus:
+          description: Reikäpaikkausten kirjaus.
+          post:
+            is: [secured]
+            description: Uuden reikäpaikkauksen kirjaaminen.
+            body:
+              application/json:
+                example: !include examples/reikapaikkausten-kirjaus-request.json
+                schema: reikapaikkausten-kirjaus-request
+            responses:
+              200:
+               body:
+                 application/json:
+                   example: !include examples/onnistunut-kirjaus-response.json
+                   schema: kirjaus-response
+              400:
+               body:
+                 application/json:
+                   example: !include examples/virhe-response.json
+                   schema: virhe-response
+              500:
+               body:
+                 application/json:
+                   example: !include examples/virhe-response.json
+                   schema: virhe-response
+    /reikapaikkaus:
+              description: Reikäpaikkausten päivitys.
+              put:
```

Comment:
Voisko molemmat post: ja put: olla saman /reikapaikkaus: osion alla? Tuota pathia ei varmaan tarvi duplikaattina?

Reply:
Joo, ei kannata toistaa polkua. Ennen kuin päivität, ota pohjalle 1.0-version api.raml-tiedosto. Pitäis löytyä developista.

Reply:
Joo. Nyt osasin varmaan 1.0 versioon tehdä sen oikein. Ainakin tuo html näyttää ihan oikein, kun generoin sen api docin. :)